src/entities/concretes/elements/touchable/flag.py

```diff
@@ -14,6 +37,42 @@ def __init__(
             position,
             elements[ElementType.FLAG][element_sub_type],
             x_rect_percent=(
-                0.11 if element_sub_type == ElementSubType.FLAG_SUPPORT else 1
+                0.145 if element_sub_type == ElementSubType.FLAG_SUPPORT else 1
             ),
         )
+        self.__points = self.__get_points()
+
+    def __get_points(self) -> int:
```

Comment:
Chaining of conditionals can be avoided if constants are handled in lists

by example:

RANGES = [
        (240, 299),
        (300, 359),
        (360, 419),
        (420, 539),
        (540, 659),
        (660, float('inf'))  # Usamos 'inf' para indicar que no hay límite superior
    ]
    
    SCORES = [
        2000,  # SCORE_FIRST_RANGE
        1000,  # SCORE_SECOND_RANGE
        800,   # SCORE_THIRD_RANGE
        400,   # SCORE_FOURTH_RANGE
        200,   # SCORE_FIFTH_RANGE
        100    # SCORE_FINAL_RANGE
    ]